OpenDreamRuntime/Objects/Types/DreamList.cs

```diff
@@ -1263,6 +1271,91 @@ public IEnumerable<DreamObjectTurf> GetTurfs() {
     }
 }
 
+// mob.contents, obj.contents list
+public sealed class MovableContentsList(DreamObjectDefinition listDef, DreamObjectMovable movable) : DreamList(listDef, 0) {
+    private readonly DreamObjectMovable _movable = movable;
+    public override DreamValue GetValue(DreamValue key) {
+        if (!key.TryGetValueAsInteger(out var index))
+            throw new Exception($"Invalid index into movable contents list: {key}");
+
+
+        if (index < 1 || index > _movable.ChildCount)
+            throw new Exception($"Out of bounds index on movable contents list: {index}");
+
+
+        using (var childEnumerator = _movable.ChildEnumerator) {
```

Comment:
It should be noted that `ChildEnumerator` is an enumerator of a `HashSet` which has no defined order. This means the order could theoretically change at any time, breaking this.

I don't think this is a new bug so it doesn't really need to be worried about in this PR.